### PR TITLE
fix: apply 1st design QA items for RTL

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Fixed several UI issues mentioned in https://sendbird.slack.com/archives/C0794CS06EM/p1718940053725789

### CODE REVIEW

File: src/modules/ChannelSettings/components/UserPanel/user-panel.scss [hunk 1]
```diff
@@ -32,5 +32,10 @@
   .sendbird-channel-settings-accordion__footer {
     padding-left: 14px;
     padding-top: 14px;
+    padding-bottom: 14px;
+    @include themed() {
+      background-color: t(bg-0);
+      border-bottom: 1px solid t(on-bg-4);
+    }
```
Comment (AhyoungRyu) [inline src/modules/ChannelSettings/components/UserPanel/user-panel.scss:39]: Not related to RTL support actually but just fixed ;) 
**as-is**
Screenshot 2024-06-24 at 5 36 51 PM

**to-be**
Screenshot 2024-06-24 at 5 34 53 PM
Screenshot 2024-06-24 at 5 34 42 PM

File: src/ui/MessageSearchItem/index.scss [hunk 2]
```diff
@@ -71,7 +71,7 @@
   }
   .sendbird-message-search-item__left {
     @include themed() {
-      border-left: 4px solid t(primary-3);
+      border-inline-start: 4px solid t(primary-3);
```
Comment (AhyoungRyu) [inline src/ui/MessageSearchItem/index.scss:74]: **as-is**
Screenshot 2024-06-24 at 5 37 36 PM
**to-be**
Screenshot 2024-06-24 at 5 37 54 PM

File: src/ui/ThreadReplies/index.scss [hunk 3]
```diff
@@ -88,4 +88,6 @@
 .sendbird-ui-thread-replies__icon {
   position: relative;
   display: inline-flex;
+  /*rtl:raw:
+  transform: rotate(180deg);*/
```
Comment (AhyoungRyu) [inline src/ui/ThreadReplies/index.scss:92]: **as-is**
Screenshot 2024-06-24 at 5 38 55 PM

**to-be**
Screenshot 2024-06-24 at 5 38 24 PM

**Review by chrisallo — APPROVED** [review]
lgtm!

**Review by HoonBaek — APPROVED** [review]
LGTM

**Pull request merged**